pkg/services/accesscontrol/acimpl/service.go

```diff
@@ -106,19 +108,36 @@ func (s *Service) GetUserPermissions(ctx context.Context, user *user.SignedInUse
 	return s.getCachedUserPermissions(ctx, user, options)
 }
 
-func (s *Service) getUserPermissions(ctx context.Context, user *user.SignedInUser, options accesscontrol.Options) ([]accesscontrol.Permission, error) {
+func (s *Service) getUserPermissions(ctx context.Context, user identity.Requester, options accesscontrol.Options) ([]accesscontrol.Permission, error) {
 	permissions := make([]accesscontrol.Permission, 0)
 	for _, builtin := range accesscontrol.GetOrgRoles(user) {
 		if basicRole, ok := s.roles[builtin]; ok {
 			permissions = append(permissions, basicRole.Permissions...)
 		}
 	}
 
+	namespace, identifier := user.GetNamespacedID()
+	if namespace != authn.NamespaceUser &&
+		namespace != authn.NamespaceServiceAccount &&
+		namespace != identity.NamespaceRenderService {
+		return permissions, nil
+	}
+
+	userID, err := strconv.ParseInt(identifier, 10, 64)
```

Comment:
question: does the render service have a user ID?

Reply:
it can have or it can not have :(

I think we need to fine tune the render client in a separate PR to distinguish when it's returning a user ID or the service itself